src/config/settings.js

```diff
@@ -1,35 +1,35 @@
-import { LOCAL_API_HOST } from './api';
+import { CONTEXTS } from './contexts';
+import { REACT_APP_MOCK_API } from './env';
 
-export const MAX_INPUT_LENGTH = 5000;
-export const MAX_ROWS = 10;
 export const DEFAULT_LANG = 'en';
-export const DEFAULT_MODE = 'student';
-export const TEACHER_MODE = 'teacher';
 
+const LOCAL_API_HOST = 'localhost:3000';
+
+// TODO: Test.
```

Comment:
Remove.

Reply:
You mean remove `// TODO: Test.` or `const LOCAL_API_HOST = 'localhost:3000';` ?